WORKSPACE

```diff
@@ -26,8 +27,26 @@ http_archive(
     ],
 )
 
+# bb-storage is downloaded to separately to apply patches to gazelle.
+# Then the real `com_github_buildbarn_bb_storage` is downloaded through `go_dependencies.bzl`.
+git_repository(
+    name = "com_github_buildbarn_bb_storage_patches",
+    remote = "https://github.com/buildbarn/bb-storage",
+    commit = "ece87ab6dc2a9e1e592d2032f5a02c3694765cfc",
```

Comment:
This is now something we need to manage. :(

Reply:
Should `http_archive` be used to download a source archive instead? If not, I'm happy for you to merge the PR.

Reply:
Good point! `http_archive` is better :)

Reply:
Done